widgets/obs-widget/src/App.tsx

```diff
@@ -55,10 +55,10 @@ export default class App extends Component<IAppProps, IAppState> {
       algorithm,
       whitelist,
     } = this.props.widgetSettings;
-    const rankingApiUrl = `${apiUrl}/ranking/${asset}:${recipientAddress}/${algorithm}/`;
-    const whitelistQueryParam = whitelist ? `?whitelist=${whitelist}` : '';
+    const rankingApiUrl = `${apiUrl}/ranking/${algorithm};asset=${asset};context=${recipientAddress}/`;
```

Comment:
Why use `;` instead of `&` to split params?

Reply:
There are not query params. Example url looks like this:
```
https:/

/api-staging.userfeeds.io
/ranking
/betweenblocks;maxBlockNumber=2000000;minBlockNumber=1000000;context=0x0406735fc1a657398941a50a0602eddf9723a6c8;asset=ropsten
/filter_whitelist;whitelist=0xdf0e37dacfb5c82c86c8a0d6742ddda4112b9c25
/?debug
```